aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -46,6 +46,14 @@ else
 	curl --silent -H "Content-Type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d '{"name":"core-user-rule","description": "Core User is Admin of all containers","role":"administrator","resources":{"containers":["*"],"images":["*"],"volumes":["*"],"networks":["*"]},"accessors":{"users":["core"]}}' $WEB_URL/adminrules
 fi
 
+EXISTING_LABEL=$(curl --write-out %{http_code} --silent --output /dev/null -H "$HEADER: Bearer $TOKEN" $WEB_URL/settings/labels/production%20approved)
+
+if [[ "$EXISTING_LABEL" == "200" ]]; then
+	log "production approved label exists..."
+else
+	curl --silent -H "Accept: application/json" -H "Content-type: application/json" -H "$HEADER: Bearer $TOKEN" -X POST -d '{"name": "production approved", "description": "production approved images"}' $WEB_URL/settings/labels
```

Comment:
A policy must be created to:
1. Block all images from running
2. allow images that have "production approved" labels to run

FD will handle associating images to the "production approved" label. However, you need to create the policy along with the label.